packages/create-remix/templates/netlify/server.js

```diff
@@ -0,0 +1,7 @@
+import { createRequestHandler } from "@remix-run/netlify";
+import * as build from "@remix-run/server-build";
```

Comment:
Let's change this to `@remix-run/dev/server-build` and get rid of the separate package.